components/automate-ui/src/styles.scss

```diff
@@ -234,3 +234,95 @@ input {
     border-radius: $global-radius $global-radius 0 0;
   }
 }
+
+// Angular Material Overrides
+.chef-control-menu-x {
+  position: absolute;
+  margin-top: 35px;
+  right: -48px;
+  margin-right: 35px;
+}
+
+.three-dot-column {
+  position: relative;
+}
+
+.mat-option {
+  font-family: 'Muli', Helvetica, Arial, sans-serif;
+
+  &.mat-active,
+  &:hover:not(.mat-option-disabled),
+  &:focus:not(.mat-option-disabled) {
+    background: $chef-primary-bright;
+    color: $chef-white;
+  }
+
+  &:hover:not(.mat-option-disabled):not(.mat-active) {
+    background: lighten($chef-primary-bright, 15%);
+  }
+}
+
+[panelclass="chef-control-menu-x"] {
+  text-align: right;
+
+  &:focus .mat-select-trigger {
+    outline: -webkit-focus-ring-color auto 5px;
+  }
+
+  .mat-select-trigger {
+    display: inline-block;
+    position: absolute;
+    right: 10px;
+    top: calc(50% - 16px); // subtracting half the height
+    border-radius: var(--chef-border-radius);
+    background-color: var(--chef-white);
+    width: 32px;
+    height: 32px;
+    cursor: pointer;
+
+    &:after {
+      content: '...';
+      position: absolute;
+      color: var(--black);
+      bottom: 35%;
+      left: 23%;
+      font-size: 20px;
+    }
+
+    &:hover {
+      background-color: var(--chef-badge-primary-background);
+
+      &:after {
+        color: var(--chef-badge-primary-color);
+      }
+    }
+  }
+
+  &[aria-owns] {  // hacky alternative to active
```

Comment:
The comment gives me only a vague idea of justification; might you have a URL reference you could add here?

Reply:
I've updated the comment so it makes a little more sense.  Ultimately, I noticed "aria-owns" being applied and hooked in on it.  

`aria-owns` by definition tells assistive technology that another element being placed on the screen should be treated as a child element of the current element we're interacting with (https://developers.google.com/web/fundamentals/accessibility/semantics-aria/aria-labels-and-relationships#aria-owns)

I'm ultimately utilizing it out of scope because when `aria-owns` is added, we know that the dropdown panel is open.